tests/unit/amo/api/test_users.js

```diff
@@ -79,6 +81,30 @@ describe(__filename, () => {
       await editUserAccount(params);
       mockApi.verify();
     });
+
+    it('sends a FormData body when a picture file is supplied', async () => {
+      const editableFields = {
+        biography: 'I am a cool tester.',
+      };
+      const picture = new File([], 'image.png');
+      const params = getParams({ picture, ...editableFields });
+
+      const expectedBody = new FormData();
+      expectedBody.set('biography', editableFields.biography);
+      expectedBody.set('picture_upload', picture);
+
+      mockApi.expects('callApi')
+        .withArgs(sinon.match(({ body }) => {
+          return deepEqual(
+            Array.from(body.entries()),
+            Array.from(expectedBody.entries())
+          );
+        }))
+        .returns(mockResponse());
```

Comment:
Why is this returning an empty response?

Reply:
I do not test/verify the response here.